pkg/manager/manager_test.go

```diff
@@ -52,16 +53,6 @@ import (
 )
 
 var _ = Describe("manger.Manager", func() {
-	var stop chan struct{}
-
-	BeforeEach(func() {
-		stop = make(chan struct{})
```

Comment:
Rather than creating and canceling a context in each testcase, can we just create the ctx and cancel in the BeforeEach and `cancel` in the `AfterEach`?

Reply:
The race detector will complain about using the context concurrently, so I changed it that way